recipes/dipy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.11.0" %}
+
+package:
+  name: dipy
+  version: {{ version }}
+
+source:
+  fn: dipy-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/d/dipy/dipy-{{ version }}.tar.gz
+  md5: 7723e8994d31e910d47b7a0cf064a3d8
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - nibabel
+    - cython
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - nibabel
+    - cython
```

Comment:
Good catch. Probably not.

On Thu, May 12, 2016 at 7:14 AM, Filipe <notifications@github.com> wrote:

> In recipes/dipy/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/308#discussion_r63027341>
> :
>
> > +build:
> > +    number: 0
> > +    script: python setup.py install
> > +
> > +requirements:
> > +  build:
> > +    - python
> > +    - numpy x.x
> > +    - nibabel
> > +    - cython
> > +  run:
> > +    - python
> > +    - numpy x.x
> > +    - scipy
> > +    - nibabel
> > +    - cython
>
> Do you need cython as a run dependency?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/308/files/11806b214f840b1bb21af55204299c8d7a715980#r63027341>
>